app/web/package.json

```
@@ -1,155 +1,96 @@
 {
-  "name": "couchers-react",
+  "name": "web",
   "version": "0.1.0",
   "private": true,
+  "scripts": {
+    "build-storybook": "build-storybook",
+    "build": "next build",
+    "dev": "next dev",
+    "start": "yarn dev",
```

```suggestion
    "start": "next dev",
```
We could combine these to mimic `start` to be starting the development server like before with CRA?

---

I kept both so people used to the old way can use start, but people who know Next uses dev can use that.

---

Ahh fair, guess it's not really a big deal, just curious why we repeat the command but the convention from the Next docs makes sense